index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Apple</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/1ba545dca5.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body class="main-container">
+    <div class="hero-container">
+        <header class="hero">
+
+            <nav class="nav-container">
+                <div class=" d-flex center-flex">
+                    <div class="apple-logo">
+                        <a href="#"><img src="images/apple-logo.svg" alt=""></a>
+                    </div>
+
+                    <div class="navbar">
+                        <a class="" href="#">Mac</a>
+                        <a class="" href="#">iPod</a>
+                        <a class="" href="#">iPhone</a>
+                        <a class="" href="#">iPad</a>
+                        <a class="" href="#">iTunes</a>
+                        <a class="" href="#">Support</a>
+                    </div>
```

Comment:
- [ ] There should be 8 items + the search bar. You missed the store.

- [ ] Also the layout of the first and last elements is different (they all have the same width). Kindly address that.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91103970-279cb480-e643-11ea-85fb-9a38cf09bcf2.png) |![image](https://user-images.githubusercontent.com/61479648/91103995-32efe000-e643-11ea-9c77-ea2d7015e597.png) |